tests/integration/test_atleast_nd.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+from legate.core import LEGATE_MAX_DIM
+
+
+def _check(a, routine, sizes):
+    b = getattr(np, routine)(*a)
+    c = getattr(num, routine)(*a)
+    is_equal = True
+    err_arr = [b, c]
+
+    if len(b) != len(c):
+        is_equal = False
+        err_arr = [b, c]
+    else:
+        for each in zip(b, c):
+            if not np.array_equal(*each):
+                err_arr = each
+                is_equal = False
+                break
+    print_msg = f"np.{routine}({sizes})"
+    assert is_equal, (
+        f"Failed, {print_msg}\n"
+        f"numpy result: {err_arr[0]}\n"
+        f"cunumeric_result: {err_arr[1]}\n"
+        f"cunumeric and numpy shows"
+        f" different result\n"
+    )
+    print(f"Passed, {print_msg}, np: {b}" f", cunumeric: {c}")
```

Comment:
Connect these two strings

Reply:
I was suggesting you do something like this:

```
print(f"Passed, {print_msg}, np: {b}, cunumeric: {c}")
 ```

Is there a reason to have this as two strings?